rust/src/action/mod.rs

```diff
@@ -332,6 +332,25 @@ impl MetaData {
     }
 }
 
+impl TryFrom<DeltaTableMetaData> for MetaData {
```

Comment:
drive by refactor, makes more sense to keep all `MetaData` impl in the same place.